storage/client_range_test.go

```diff
@@ -146,7 +146,7 @@ func TestRejectFutureCommand(t *testing.T) {
 	if err != nil {
 		t.Fatal(err)
 	}
-	if v := val.GetInteger(); v != 15 {
+	if v := val.MustGetInteger(); v != 15 {
```

Comment:
all uses of `MustGetInteger` are in `storage/`, so it looks like a helper in the storage tests makes it obsolete as a method on `Value`.